client/src/App.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-app id="app">
+    <div id="main-column">
+      <Header />
+      <router-view />
+    </div>
+  </v-app>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import Header from '@/components/header.vue'
+
+@Component({
+  components: { Header }
+})
+export default class App extends Vue {}
+</script>
+<style lang="stylus">
```

Comment:
As mentioned above, Vuetify 2 is now built using SASS/SCSS, so you might want to change your local styles to that (instead of Stylus).